.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Global Owners
-* @amisevsk @l0rd @rhopp @nickboldt @vparfonov
+* @sleshchenko @amisevsk @l0rd @rhopp @nickboldt @vparfonov
```

Comment:
Well, I'm not sure how this list was formed. 
@l0rd @rhopp @nickboldt @vparfonov Do you still want to be code owners here?